### PR TITLE
Added 2 sample programs in C.

### PR DESCRIPTION
**Pull request by AceKnight**
-A Simple loop that prints number from 1 to 10.
-A Program the takes a number as an input and outputs if it's an Even or an Odd number.
-A Program the takes a number as an input and outputs if it's a Prime Number or Not.

I'm sorry if i did anything wrong. I'm new to this

### CODE REVIEW

**Comment by jrg94** [comment]
Awesome! These are all great programs, but before we can merge them, I'd like to get a formal description of the problems you're solving in the contributing doc. Could you try to take a crack at that? Otherwise, I might ask @jpvg10 to consider taking some time to do it. Then, we'll merge your programs. 

Also, since @chrboe is the resident C expert, I'll have them take a look at your code! Hope that sounds good. 😄

**Comment by AceKnight** [comment]
> Awesome! These are all great programs, but before we can merge them, I'd like to get a formal description of the problems you're solving in the contributing doc. Could you try to take a crack at that? Otherwise, I might ask @jpvg10 to consider taking some time to do it. Then, we'll merge your programs.

 I've never done this before. I'm really sorry but I don't know how to do that 😟 .
are you talking about updating the readme file ?

**Comment by AceKnight** [comment]
I've 
-Fixed Formatting
-Removed platform specific function: getch()
-Removed the simple loop program.
-Added description of the Programs in the Contributing document.

Please take a look and tell me if I did anything wrong. 
I've made commit to the fork. Does the Pull request update automatically?

**Comment by jrg94** [comment]
Hey @AceKnight! I like what you've done so far. Can you pull the latest changes from this repo? We have updated the way we handle rule descriptions, so the two sections you added today would have to go into their own files.

Let me know if I can help you through this process!

**Comment by jrg94** [comment]
By the way, the pull request does update automatically. However, your fork does not update automatically. You'll need to merge in the latest changes before we can merge your pull request.

**Comment by AceKnight** [comment]
I don't know what i did just know, My CONTRIBUTING.md file is all messed up now.

**Comment by jrg94** [comment]
Yes! So, you basically merged all those conflicts which are denoted by the `<<<<<<<<head` lines in your file. You'll need to go through and decide which sections you want and which you don't. In other words, you'll have to remove either the section between `<<<<<head` and `======` or `==========` and `>>>>>>>>>>>>>`.

I can give more details when I have a moment.

**Comment by AceKnight** [comment]
I hope everything is fixed now

**Comment by jrg94** [comment]
This is excellent! Nice work getting things working. I'll let @TheRenegadeCoder/core look over it (specifically @Noah11012 who has to approve this). Otherwise, I'm very happy with what you've accomplished.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -153,6 +153,8 @@ each link to read more about their rules.
 -   [Quine Rules][9]
 -   [Reverse a String Rules][8]
 -   [Roman Numeral Conversion][26]
```
Comment (jrg94) [inline CONTRIBUTING.md]: While you're here, can you update these last three to include the word `Rules` at the end like the rest of the list?

File: docs/even-odd.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Create a file called Even Odd using the naming
+convention appropriate for your language of choice.
+
+Write a sample program which accepts an integer on the command line and
+outputs if the integer is Even or Odd.
+
+## Testing
+
+Verify that the actual output matches the expected output. See the
```
Comment (jrg94) [inline docs/even-odd.md]: I think for both this and prime you could include a table of example inputs and outputs like reverse a string or roman numeral. That way we can explore all the possible test cases. This one should include one odd number, one even number, zero, and a negative number.

File: docs/prime-number.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Create a file called Prime Number using the naming
+convention appropriate for your language of choice.
+
+Write a sample program which accepts an integer on the command line
+and outputs if the integer is a Prime number or not.
+
+## Testing
+
+Verify that the actual output matches the expected output. See the
```
Comment (jrg94) [inline docs/prime-number.md]: For prime, we could include a table that has the same test cases as even/odd. One prime number, one not prime number, zero, and a negative number. It may be good to include a test case for empty input as well.

File: archive/c/c/even-odd.c [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+int a;
+printf("Enter a Number: ");
+scanf("%d", &a);
+
+if(a%2==0)
+    printf("%d is a Even.", a);
+else
+    printf("%d is Odd.", a);
+
+return 0;
```
[the remark this answers is not in the document]
Comment (jrg94) [reply archive/c/c/even-odd.c]: Agreed! All your code in the main block (between the braces) should be indented a level deeper.

**Comment by AceKnight** [comment]
Hey, I fixed the indentation. Sorry, I couldn't do the Table in the docs 😟

**Comment by jrg94** [comment]
I don't mean to overload you! 😄 I'm just trying to offer some helpful advice. Also, we have pretty high standards for this repo because we want to provide the best possible solutions for folks who might want to see code implemented in various ways. 

That said, we really appreciate your contribution, and we hope you stick around.

**Comment by AceKnight** [comment]
Tbh I feel really stupid 😂 . You guys have been great and I couldn't even fix this simple issue. I really hope i did everything correctly this time. 
I feel happy to contribute to this project and would like to help out in the future as well. 
I learned alot. Thanks ^^

**Comment by jrg94** [comment]
Don’t sweat it! A large chunk of my time is devoted to being a student, so I understand where you’re coming from! I also have 40 of my own students, so I’ve learned to be very patient for their sake 😄 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jrg94** [comment]
Welp! I said one thing and did another.. accidentally merged instead of squashing. Oh well! It's late. 😆
